.github/workflows/stardis-tests.yml

```diff
@@ -0,0 +1,79 @@
+name: Stardis Tests
```

Comment:
I think that is `STARDIS`, might have to confirm with others, or I will just put lowercase `stardis` for now.

Reply:
Updated it